package.json

```diff
@@ -32,51 +32,45 @@
   },
   "homepage": "https://github.com/arawa/workspace#readme",
   "devDependencies": {
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/preset-env": "^7.16.11",
+    "@nextcloud/babel-config": "^1.0.0",
     "@nextcloud/browserslist-config": "^2.2.0",
-    "@nextcloud/eslint-config": "^4.0.0",
-    "@nextcloud/eslint-plugin": "^2.0.0",
-    "@nextcloud/webpack-vue-config": "^3.0.0",
+    "@nextcloud/eslint-config": "^6.1.2",
+    "@nextcloud/stylelint-config": "^2.1.2",
+    "@nextcloud/webpack-vue-config": "^4.3.2",
     "@vue/test-utils": "^1.3.0",
-    "babel-eslint": "^10.1.0",
     "babel-loader": "^8.2.3",
-    "chai": "^4.3.4",
-    "css-loader": "^5.2.7",
-    "eslint-webpack-plugin": "^2.6.0",
+    "chai": "^4.3.6",
+    "css-loader": "^6.5.1",
+    "eslint": "^8.8.0",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-webpack-plugin": "^3.1.1",
     "jsdom": "^19.0.0",
     "jsdom-global": "^3.0.2",
-    "mocha": "^9.2.0",
-    "mochapack": "^2.1.2",
+    "linkifyjs": "^3.0.5",
+    "mochapack": "^2.1.4",
+    "node-polyfill-webpack-plugin": "^1.1.4",
     "postcss-html": "^1.3.0",
     "prettier": "^2.5.1",
+    "style-loader": "^3.3.1",
     "stylelint": "^14.3.0",
-    "stylelint-config-recommended-scss": "^4.2.0",
+    "stylelint-config-recommended-scss": "^5.0.2",
     "stylelint-config-recommended-vue": "^1.1.0",
-    "stylelint-scss": "^3.21.0",
+    "stylelint-scss": "^4.1.0",
+    "stylelint-webpack-plugin": "^3.1.1",
     "url-loader": "^4.1.1",
-    "vue-loader": "^15.9.6",
-    "vue-template-compiler": "^2.6.12",
-    "webpack": "^4.46.0",
+    "vue-loader": "^15.9.8",
+    "vue-template-compiler": "^2.6.14",
+    "webpack": "^5.67.0",
```

Comment:
https://github.com/nextcloud/webpack-vue-config/blob/dc4a96cba71d3b47cec61257a441c551c5c1e6cc/package.json#L29-L46

Reply:
My PR was done and working (beside the tests), have a look at it :)

Reply:
It's strange ! Because, without these dependencies, the build doesn't work...

I look at the workflow-templates/node.yml

Reply:
Your CI on 12 is using npm 6, you should also not use `npm i` for CI, but `npm ci.`

Reply:
https://github.blog/2021-02-02-npm-7-is-now-generally-available/

> ## Peer dependencies
> Automatically installing peer dependencies is an exciting new feature introduced in npm 7. In previous versions of npm (4-6), peer dependencies conflicts presented a warning that versions were not compatible, but would still install dependencies without an error. npm 7 will block installations if an upstream dependency conflict is present that cannot be automatically resolved.
> 
> You have the option to retry with --force to bypass the conflict or --legacy-peer-deps command to ignore peer dependencies entirely (this behavior is similar to versions 4-6).
> 
> Since many packages in the ecosystem have come to rely on loose peer dependencies resolutions, npm 7 will print a warning and work around most peer conflicts that exist deep within the package tree, since you can’t fix those anyway. To enforce strictly correct peer dependency resolutions at all levels, use the --strict-peer-deps flag.

Reply:
https://github.com/nextcloud/viewer/blob/8f0f5164192e9576e45e70aeb310936424b02520/package.json#L65-L72

Reply:
> https://github.com/nextcloud/webpack-vue-config/blob/dc4a96cba71d3b47cec61257a441c551c5c1e6cc/package.json#L29-L46

Oooh cool !
I will remove the dependencies I added a few minutes ago...

Reply:
> Your CI on 12 is using npm 6, you should also not use `npm i` for CI, but `npm ci.`

Okay ! What is the different ?

Reply:
> https://github.blog/2021-02-02-npm-7-is-now-generally-available/
> 
> > ## Peer dependencies
> > Automatically installing peer dependencies is an exciting new feature introduced in npm 7. In previous versions of npm (4-6), peer dependencies conflicts presented a warning that versions were not compatible, but would still install dependencies without an error. npm 7 will block installations if an upstream dependency conflict is present that cannot be automatically resolved.
> > You have the option to retry with --force to bypass the conflict or --legacy-peer-deps command to ignore peer dependencies entirely (this behavior is similar to versions 4-6).
> > Since many packages in the ecosystem have come to rely on loose peer dependencies resolutions, npm 7 will print a warning and work around most peer conflicts that exist deep within the package tree, since you can’t fix those anyway. To enforce strictly correct peer dependency resolutions at all levels, use the --strict-peer-deps flag.

Yes, I heard of it the last week. I think it's very cool this feature :)

Reply:
https://docs.npmjs.com/cli/v8/commands/npm-ci